### ISSUE TITLE
New Problems: Gradient ops based problems

### ISSUE THREAD

**Issue by komaksym** [Open-Deep-ML/DML-OpenProblem#479]
I think it would be cool to also have the core gradient operation based problems like:
* Gradient Clipping
* Gradient Accumulation
* Gradient Checkpointing

Can work on these too @moe18

**Comment by moe18** [comment]
Sounds good, are you just suggesting or would you like to work on this, either is good
